src/lib/eZ/RichText/Resources/stylesheets/xhtml5/edit/docbook.xsl

```diff
@@ -85,13 +88,30 @@
     </xsl:choose>
   </xsl:template>
 
+  <xsl:template name="conditionallyAddIdAttribute">
```

Comment:
It's not actually conditionally adding ID attribute. It's changing the ID if a second one exists. We probably need a different name.

Reply:
@Steveb-p 
Renamed it to `addUniqueIdAttribute` in 104897d19dcfba421f178396f8d3088f108538a7.
ok ?